bin/find.py

```diff
@@ -112,7 +112,7 @@ def main(args):
     ap.add_argument('-maxdepth', '--maxdepth',
                     metavar='n',
                     nargs='?',
-                    default=sys.maxint,
+                    default=(2 ** 30),
```

Comment:
`sys.maxsize` (means a slightly different thing than Python 2's `sys.maxint`, but works well as an "almost infinity" number)

Alternatively, default to `None`, and skip the depth check in that case.